spec/classes/config_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'gitlab_ci_runner::config' do
```

Comment:
can you keep those tests in the other spec file? private classes are not supposed to be tested directly.

Reply:
Could you please point me to the relevant documentation for this? For me it makes more the sense to extract the tests from the main tests file simply to keep it small and test resource where they are actually defined and used. If I wouldn't do this, modules with a lot of private class (and resources) would have a `init_spec.rb` with 100000s of lines.

Reply:
As discussed on Slack, this can stay as it is.